helm/templates/workflow-pvc.yaml

```diff
@@ -9,4 +9,8 @@ spec:
   resources:
     requests:
       storage: 5Gi  # Match this to your PV's storage
-  storageClassName: {{ if eq .Values.ENV "dev" }}""{{ else }}"csi-cephfs-sc"{{ end }}
+  {{ if eq .Values.env "dev" }}
+  storageClassName: ""
+  {{ else }}
+  storageClassName: "csi-cephfs-sc"
```

Comment:
Does it make sense to get the storage class name from the `values.yaml`, based on the `env`? It seems like we do that in `minio-pvc.yaml`? 

> {{ (index .Values.minio.persistence .Values.env).storageClassName | quote }}